### PR TITLE
[Merged by Bors] - fix: variables in TuringMachine.lean

### PR DESCRIPTION
**Pull request by kim-em**
This is work by @digama0 that I'm backport from the lean-pr-testing-4814 branch.

### CODE REVIEW

File: Mathlib/Computability/TuringMachine.lean [hunk 1]
```diff
@@ -932,26 +931,29 @@ def Machine [Inhabited Λ] :=
...
 local notation "Cfg₀" => Cfg Γ Λ  -- Porting note (#10750): added this to clean up types.
 
-instance Cfg.inhabited : Inhabited Cfg₀ :=
-  ⟨⟨default, default⟩⟩
-
 variable {Γ Λ}
+variable [Inhabited Λ]
+
+section
+variable [Inhabited Γ]
```
Comment (grunweg) [inline Mathlib/Computability/TuringMachine.lean:954]: This looks funny - surely you don't need both this and line 951?

**Review by grunweg — COMMENTED** [review]
Looks innocuous. Can you fix the build failure, please?

**Review by sgouezel — COMMENTED** [review]
bors r+

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**